app/templates/src/MyService.ts

```diff
@@ -0,0 +1,35 @@
+import { InputSchema, OutputConfiguration, ServiceDefinition, SimpleMap, WebServiceError } from '@sirensolutions/web-service-interface';
+import axios from 'axios';
+
+export default class MyService extends ServiceDefinition<{ auth_token: string }> {
```

Comment:
I think it would be worth adding a question about what the user would want their first service to be called and then injecting it into here (and as the filename when bootstrapping).

Then the user is basically up and running once it's bootstrapped?